satpy/etc/readers/insat3d_img_l1b_h5.yaml

```diff
@@ -0,0 +1,162 @@
+reader:
+  name: insat3d_img_l1b_h5
```

Comment:
Is there a version of this L1B data that isn't HDF5?

Reply:
Yes, geotiff also according to the website